packages/blockchain/src/index.ts

```diff
@@ -483,6 +487,12 @@ export default class Blockchain implements BlockchainInterface {
     dbOps.push(DBOp.set(DBTarget.CliqueSignerStates, rlp.encode(formatted)))
 
     await this.dbManager.batch(dbOps)
+    // Output active signers for debugging purposes
+    let i = 0
+    for (const signer of this.cliqueActiveSigners()) {
+      debug(`Clique signer [${i}]: ${signer}`)
+      i++
+    }
```

Comment:
nice little es6 syntax destructuring trick for getting the index of an iteration in a for-of loop:
```suggestion
    for (const [i, signer] of this.cliqueActiveSigners().entries()) {
      debug(`Clique signer [${i}]: ${signer}`)
    }
```

Reply:
Wow, cool 😄